### PR TITLE
Submission Sucess

### PR DESCRIPTION
**Pull request by flashrv3**
Application submitted successfully page, also open to suggestions

### CODE REVIEW

File: css/application-styles.css [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+.form-application-cvfile-button {
+  height: inherit;
+  width: 248px;
+  border: 0.5px solid rgba(0, 0, 0, 1);
+  font-family: Inter;
+  font-size: 36px;
+  font-weight: 400;
+  line-height: 44px;
+  border-right: none;
```
Comment (ilona-liashenko) [inline css/application-styles.css]: I think it's worth removing border-right: none as there should be a border according to the figma layout

File: css/application-styles.css [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.form-applicaion {
+  background: #FFC0CB;
+  border: 1px solid #EDDEDE;
+  border-radius: 12px; 
+  padding: 60px 0 100px 70px;
```
Comment (ilona-liashenko) [inline css/application-styles.css]: have a look and try to fix the distance between header and the form according to the layout at the top (it should be approx 80px) and bottom (approx 174px) and the padding from the top of the form to "Job Application" title. I don't know why, but I can't see exact measures between the elements in this form, maybe you can do this:)

**Review by ilona-liashenko — APPROVED** [review]

File: css/application-styles.css [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+                        'email jobrole'
+                        'labeladdress .'
+                        'address address'
+                        'cvlabel .'
+                        'cv . '
+                        'sendForm sendForm';
+  height: 80%;
+}
+
+.form-application-title {
```
Comment (elishab-dev) [inline css/application-styles.css]: please check the title positioning, in figma it's located higher

File: css/application-styles.css [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+/* ARROW ON SELECT ELEMENT*/
+
+.dropicon {
+  overflow: hidden;     
+  position: relative;
+}
+
+.dropicon:after{
+  content:"\f078 ";
+  font-family: FontAwesome;
```
Comment (elishab-dev) [inline css/application-styles.css]: I don't quite get it. the font is not imported so why do you need to use it?

File: css/application-styles.css [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+  width: 460px;
+  height: 64px;
+  padding: 0 15px;
+  border-radius: 3px;   
+  border: 1px solid #D2D2D2; 
+  appearance: none;
+}
+
+option {
+  font-family: Calibri;
```
Comment (elishab-dev) [inline css/application-styles.css]: please use the family as a fallback as well in all fonts. for example, font-family: Calibri, sans-serif

File: pages/application-green.html [hunk 6]
```diff
@@ -4,6 +4,76 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
+    <link rel="stylesheet" href="../css/normalize.css" />
+    <link rel="stylesheet" href="../css/styles.css" />
```
Comment (elishab-dev) [inline pages/application-green.html:8]: when you link the main styles, I would suggest using helper classes as well as it helps to significantly reduce the code

File: pages/application-green.html [hunk 7]
```diff
@@ -4,6 +4,76 @@
...
+          <li>
+            <button class="primary-btn mr-60">
+              <a href="book-table.html" class="caps-text f-s32">Order now</a>
+            </button>
+          </li>
+        </div>
+      </ul>
+    </nav>
+    <div class="submission-container">
+      <p class="submission-paragraph">Application submitted successfully</p>
```
Comment (elishab-dev) [inline pages/application-green.html]: there is no such a big padding/margin after the p

File: pages/application.html [hunk 8]
```diff
@@ -4,6 +4,121 @@
...
+        <label for="address" class="form-applicaion-labeladdress"
+          >Enter Address</label
+        >
+        <textarea
+          name="address"
+          id="address"
+          class="form-applicaion-address"
+        ></textarea>
+        <label for="cvfile" class="form-applicaion-cvlabel"
+          >Upload Your CV</label
```
Comment (elishab-dev) [inline pages/application.html]: there is a bigger margin after the label

**Review by elishab-dev — CHANGES_REQUESTED** [review]
hello, thank you for submitting your part for review! I noticed you haven't applied any helper classes from styles.css, is there any reason behind this? Using them would significantly reduce the code.

File: css/application-styles.css (hunk 4 above)
Comment (flashrv3) [reply css/application-styles.css]: I used this to replace the dropdown arrow on the select element and the font link was on the HTML but i decide to do in another way

File: css/application-styles.css (hunk 3 above)
Comment (flashrv3) [reply css/application-styles.css]: Thank you, for the alert i think it is fixed now.

File: css/application-styles.css (hunk 1 above)
Comment (flashrv3) [reply css/application-styles.css]: True and thanks, i had to zoom in to see that in fact there is a border, its fix right now

File: css/application-styles.css (hunk 2 above)
Comment (flashrv3) [reply css/application-styles.css]: I think is fixed, the issue was i forgot that headings has margins by default

**Review by gretamas — APPROVED** [review]

**Review by elishab-dev — APPROVED** [review]

**Pull request merged**